downstream/assemblies/platform/assembly-configure-aap-operator.adoc

```diff
@@ -6,17 +6,17 @@ ifdef::context[:parent-context: {context}]
 
 = Configuring the {OperatorPlatform} on {OCP}
 
-The central configuration for {PlaformNameShort} allows you to manage the following {PlatformNameShort} components form a single user interface:
+The platform gateway for {PlaformNameShort} allows you to manage the following {PlatformNameShort} components form a single user interface:
 
 * {ControllerNameStart}
 * {HubNameStart}
 * {EDAName}
 * {LightspeedShortName} (This feature is disabled by default, you must opt in to use it.)
 
-Before you can deploy the central configuration you need to have {OperatorPlatform} installed in a namespace. 
+Before you can deploy the platform gateway you need to have {OperatorPlatform} installed in a namespace. 
```

Comment:
I was told Platform gateway was the replacement for Gateway. Is Central configuration equivalent to platform gateway?

Reply:
Yeah I was using "Central configuration" instead of referencing Gateway UI when I didn't know the correct term. After Donna's update I updated all references